Scripts/script-CreateEmailHtmlBody.yml

```diff
@@ -83,7 +95,17 @@ args:
 - name: object
   description: Values object provided as stringified JSON
   isArray: true
+- name: replaceNotFound
```

Comment:
maybe rename to removeNotFound? replace gives the perception you will allow replacing it with something, where in fact you are just deleting it, ni?